includes/render_config.h

```diff
@@ -70,17 +70,16 @@
 //Legacy - not needed on new installs
 //#undef METATILEFALLBACK
 
-// Metatiles are much larger in size so we don't need big queues to handle large areas
+// Typical osm.org render server can render 7 metatiles per second, and has average load of 4.5 metatiles per second.
+// Queue depth of (7 - 4.5) * 86400 = 216000 moves the load from the busy part of the day to the night time
 #ifdef METATILE
 #define QUEUE_MAX (64)
 #define REQ_LIMIT (32)
-#define DIRTY_LIMIT (1000)
-
+#define DIRTY_LIMIT (216000)
 #else
 #define QUEUE_MAX (1024)
 #define REQ_LIMIT (512)
-#define DIRTY_LIMIT (10000)
-#define HASHIDX_SIZE 22123
```

Comment:
What is `HASHIDX_SIZE` and why are you removing it?

Reply:
This line is no-op.
It is used here: https://github.com/openstreetmap/mod_tile/blob/62c4a5ac7a57720182125b8aad11929dae947e46/src/request_queue.c#L39 
and redefined here: https://github.com/openstreetmap/mod_tile/blob/62c4a5ac7a57720182125b8aad11929dae947e46/includes/request_queue.h#L31 
(see includes order in https://github.com/openstreetmap/mod_tile/blob/62c4a5ac7a57720182125b8aad11929dae947e46/src/request_queue.c#L29)

Reply:
So does the hash table have 2213 bucket? or 22123 bucket? Not that either is anywhere big enough once you have a two million entry queue - that will give an average chain length of either 100 or 1000 either of which seems way too large.

Reply:
Seems like somebody wants to increase HASHIDX_SIZE from default 2213 to 22123 when METATILE is not defined. But it's so ugly and include order dependent that should be fixed.

Reply:
@apmon According to the code history, here is the initial commit to use different values for HASHIDX_SIZE:
https://github.com/openstreetmap/mod_tile/commit/da1981135b2f53626bef19e34fa1958a3062be82

But later, here after refactoring, it was broken, and now HASHIDX_SIZE is always (not obvious) is 2213:
https://github.com/openstreetmap/mod_tile/commit/707e125e3548bb57510addefe7a3f8bc13769132

Reply:
@tomhughes I've updated numbers to match ratio.
We're not using non-metatile mode, so it's not 2M, but 200k records.

@vng I've updated `request_queue.h` to not rewrite the value.